src/components/simulations/settings/create-simulation-form.tsx

```diff
@@ -27,51 +30,92 @@ export const CreateSimulationForm: FC = () => {
   })
 
   const {
+    register,
     formState: { isValid },
     getValues,
     handleSubmit,
-  } = useForm<TInitSimulationDto>({
-    resolver: zodResolver(ZInitSimulationDto),
-    values: {
-      epci: Object.entries(rates).map(([epci]) => ({ code: epci })),
-      scenario: {
-        b2_scenario: queryStates.omphale as string,
-        epcis: Object.entries(rates).reduce(
-          (acc, [epci, rates]) => {
-            acc[epci] = {
-              b2_tx_rs: rates.txRS ?? undefined,
-              b2_tx_vacance: rates.txLV ?? undefined,
-            }
-            return acc
-          },
-          {} as Record<string, TInitSimulationDto['scenario']['epcis'][string]>,
-        ),
-        projection: (queryStates.projection as number) ?? 2030,
-      },
-    },
+  } = useForm<TInitSimulationForm>({
+    resolver: zodResolver(ZInitSimulationForm),
```

Comment:
You want to be able to validate the whole object (name, scenario, epci and projection)
Here you are just validating having a name on the simulation

Even if we don't display error messages (for now) in case the form is malformed, we want to validate everything